lib/krb5/pac.c

```diff
@@ -1024,6 +1061,16 @@ _krb5_pac_sign(krb5_context context,
 				       N_("PAC have two logon names", ""));
 		goto out;
 	    }
+	} else if (p->pac->buffers[i].type == PAC_TICKET_CHECKSUM) {
+	    if (p->ticket_checksum == NULL) {
+		p->ticket_checksum = &p->pac->buffers[i];
+	    }
+	    if (p->ticket_checksum != &p->pac->buffers[i]) {
+		ret = EINVAL;
```

Comment:
Wonder if there is a more Kerberos-specific error than EINVAL, eg. `KRB5KDC_ERR_BADOPTION` or `KRB5KRB_APERR_BAD_INTEGRITY`.

Reply:
I'll go with KRB5KDC_ERR_BADOPTION as integrity sounds like checksum failure to me.